src/status.rs

```diff
@@ -225,6 +222,136 @@ pub enum StatusCode {
 
 impl StatusCode {
 
+    #[doc(hidden)]
```

Comment:
Why hidden for this and `to_u16`? That's something people are likely to want to do.